### PR TITLE
Feature Reuest: Append slug: hour, minutes, secounds

### PR DESCRIPTION
**Pull request by terrierscript**
**- Summary**
When I'm not want to use `{{slug}}`, it's case occur happen can only 1 post per one day.
This append slug hours, minutes, seconds like `{{year}}-{{month}}-{{day}}-{{hours}}-{{minutes}}-{{seconds}}`.

**- Test plan**
execute yarn test and build

**- Description for the changelog**
* Append implementation 
* Append document
* Append contributor ( if this PR is merged )

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 5fc6e1e393c7eee96a0773a98de9a728eac9a8db

https://deploy-preview-1207--cms-demo.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 5fc6e1e393c7eee96a0773a98de9a728eac9a8db

https://deploy-preview-1207--netlify-cms-www.netlify.com

File: website/site/content/docs/configuration-options.md [hunk 1]
```diff
@@ -133,6 +133,9 @@ For folder collections where users can create new items, the `slug` option speci
 - `{{year}}`: 4-digit year of the file creation date
 - `{{month}}`: 2-digit month of the file creation date
 - `{{day}}`: 2-digit day of the month of the file creation date
+- `{{hour}}`: 2-digit hour of the file creation date
+- `{{minute}}`: 2-digit minute of the month of the file creation date
+- `{{second}}`: 2-digit second of the month of the file creation date
```
Comment (erquhart) [inline website/site/content/docs/configuration-options.md]: Typos in these last two lines (extra "of the month" in each).

Comment (terrierscript) [reply website/site/content/docs/configuration-options.md]: I fixed it!

**Pull request merged**